tests/serializers/test_player.py

```diff
@@ -1,18 +1,16 @@
-"""A module for testing player serializer."""
+"""A module for testing the player serializer."""
 import pytest
 from django.contrib.auth.models import User
-
 from mars_api.models import Player
```

Comment:
What happened here then - you may have to tell isort that mars_api is a local dependency